<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="p4a-508SD5S" name="Test1" tests="1" failures="1" timestamp="25 Jul 2018 07:19:28 GMT" time="0.015" errors="0">
  <testcase name="f" time="0.004" classname="com.accenture.test.LoginTest">
    <failure type="java.lang.AssertionError" message="Failed expected [Ash] but found [Ashvin]">
      <![CDATA[java.lang.AssertionError: Failed expected [Ash] but found [Ashvin]
	at com.accenture.test.LoginTest.f(LoginTest.java:11)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- f -->
</testsuite> <!-- Test1 -->
